framework/include/meshgenerators/MeshGenerator.h

```diff
@@ -213,6 +367,252 @@ MeshGenerator::declareMeshProperty(const std::string & data_name, const T & init
   return data;
 }
 
+template <typename T>
+inline bool
+MeshGenerator::typeDoubleCheck(T temp_value, const std::string type_name)
+{
+  (void)temp_value;
```

Comment:
what is this for

Reply:
Just to get the type through template. `{void)` is to "use" it so that the compiler does not complain.

Reply:
Can you do MeshGenerator::typeDoubleCheck(T /* temp_value */, 

?